vkapi/vkapi.py

```diff
@@ -722,10 +728,24 @@ def __init__(self,
 
     root = ET.parse(registry_file).getroot()
 
+    # Set the default scope to core platform
     if platforms is None:
       platforms = ['']
+    if authors is None:
+      authors = ['']
+    if supported is None:
+      supported = ['vulkan']
+
+    # Extend the parameters if necessary
+    if VULKAN_ALL_PLATFORMS in platforms:
+      platforms = ['']
       platforms.extend(
-          [p.get('name') for p in root.findall("platforms/platform[@name]")])
```

Comment:
It looks like this was already set up so that `platforms=None` has similar behaviour to what you've implemented in `VULKAN_ALL_PLATFORMS`. Is that accurate? Are there any changes here other than using `platforms=[VULKAN_ALL_PLATFORMS]` vs `platforms=None` (and similarly for the other filters)?

I'm not opposed to using these constants--it is more explicit than `None`. I just want to make sure I'm understanding the change.